File: livingrimoire start here/LivinGrimoire java/src/Auxiliary_Modules/LGTypeConverter.java
package Auxiliary_Modules;
import LivinGrimoire.*;

public class LGTypeConverter {
    public int convertToInt(String v1){
        String temp = RegexUtil.extractRegex(enumRegexGrimoire.integer, v1);
        if (temp.isEmpty()) {return 0;}
        return Integer.parseInt(temp);
    }
    public Double convertToDouble(String v1){
        String temp = RegexUtil.extractRegex(enumRegexGrimoire.double_num, v1);
        if (temp.isEmpty()) {return 0.0;}
        return Double.parseDouble(temp);
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/auxiliary_modules/LGTypeConverter.kt
package auxiliary_modules

class LGTypeConverter {
    fun convertToInt(v1: String): Int {
        val temp: String = RegexUtil.extractRegex(enumRegexGrimoire.Integer, v1)
        return if (temp.isEmpty()) {
            0
        } else temp.toInt()
    }

    fun convertToDouble(v1: String): Double {
        val temp: String = RegexUtil.extractRegex(enumRegexGrimoire.Double_num, v1)
        return if (temp.isEmpty()) {
            0.0
        } else temp.toDouble()
    }
}